src/Controller/Admin/AssetsController.php

```diff
@@ -122,7 +127,7 @@ public function download(string $id): Response
         });
 
         $response = $this->getResponse()
-            ->withType($asset->mimetype)
+            ->withType($asset->mimetype ?? 'jpg')
```

Comment:
What if the asset doesn't have a mime type? Currently `$asset->mimetype` is nullable.